src/main/litmus-2-0-0-beta/README.md

```diff
@@ -0,0 +1,111 @@
+# litmus-2-0-0-beta
+
+![Version: 2.0.22-Beta8](https://img.shields.io/badge/Version-2.0.22--Beta8-informational?style=flat-square) ![AppVersion: 2.0.0](https://img.shields.io/badge/AppVersion-2.0.0-informational?style=flat-square)
+
+A Helm chart to install litmus portal
+
+**Homepage:** <https://litmuschaos.io>
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| rajdas98 | raj.das@mayadata.io |  |
+| ispeakc0de | shubham.chaudhary@mayadata.io |  |
+| jasstkn | jasssstkn@yahoo.com |  |
+
+## Source Code
+
+* <https://github.com/litmuschaos/litmus>
+
+## Requirements
+
+Kubernetes: `>=1.16.0-0`
+
+## Installing the Chart
+
+To install this chart with the release name `litmus-portal`:
+
+```console
+$ helm repo add litmuschaos https://litmuschaos.github.io/litmus-helm/
```

Comment:
Please update this URL with:
`helm repo add main https://charts.kubesphere.io/main`